app/views/job_completion_summaries/_details.html.erb

```diff
@@ -58,36 +61,30 @@
                       <div class="col-md-12">
                         <h6>Error Details:</h6>
                         <div class="bg-light p-3 rounded">
-                          <p><strong>Origin:</strong> <%= entry['origin'] %></p>
-                          <p><strong>Job:</strong> <%= @job_completion_summary.source_name %></p>
-                          <% if entry.dig('details', 'stack_trace').present? %>
+                          <p><strong>Message:</strong> <%= record.message %></p>
+                          <p><strong>Origin:</strong> <%= record.origin %></p>
+                          <p><strong>Job:</strong> <%= @job_completion_summary.pipeline_name %></p>
+                          <% if record.stack_trace.present? %>
                             <p><strong>Stack Trace:</strong></p>
-                            <pre class="bg-dark text-light p-2 rounded"><%= entry.dig('details', 'stack_trace') %></pre>
-                          <% end %>
+                            <% stack_trace_text = if record.stack_trace.is_a?(Array)
+                                                    record.stack_trace.join("\n")
+                                                  else
+                                                    record.stack_trace.strip
+                                                  end %>
+                            <pre class="bg-dark text-light p-2 rounded"><%= stack_trace_text %></pre>
+                        <% end %>
                         </div>
                       </div>
                     </div>
                   <% end %>
-                  <% if entry['details'].present? %>
-                    <div class="row mt-3">
-                      <div class="col-md-12">
-                        <h6>Additional Details:</h6>
-                        <pre class="bg-light p-3 rounded">
-                          <% begin %>
-                            <%= JSON.pretty_generate(entry['details']) %>
-                          <% rescue %>
-                            <%= entry['details'].to_s %>
-                          <% end %>
-                        </pre>
-                      </div>
-                    </div>
-                  <% end %>
                   <div class="row mt-3">
                     <div class="col-md-6">
                       <strong>Occurred At:</strong>
-                      <% begin %>
-                        <%= Time.zone.parse(entry['timestamp']).strftime('%Y-%m-%d %H:%M:%S') %>
-                      <% rescue %>
+                      <% if record.updated_at.present? %>
+                        <%= record.updated_at.strftime('%Y-%m-%d %H:%M:%S') %>
+                      <% elsif record.created_at.present? %>
+                        <%= record.created_at.strftime('%Y-%m-%d %H:%M:%S') %>
+                      <% else %>
```

Comment:
You can use `ecord.updated_at.to_fs(:medium)` I think, look at the datetime_formats.rb file